CONTRIBUTING.md

```diff
@@ -36,28 +43,27 @@ The following is a summary of the ideal contribution flow. Please, note that Pul
     └───────────────────────┘
 ```
 
-## Code of Conduct
-AsyncAPI has adopted a Code of Conduct that we expect project participants to adhere to. Please [read the full text](./CODE_OF_CONDUCT.md) so that you can understand what sort of behaviour is expected.
+Issues and pull requests without activity from the creator within 14 days will be automatically closed. However, closure does not mean rejection. If you wish to revisit a closed issue or pull request, feel free to open a new one referencing the closed item.
```

Comment:
What if maintainers fail to acknowledge the issue within 14 days?

Reply:
I hope that never happens

so "automatically" I mean that triager or committer may automatically close the issue if it is not active

on our side

we could have maybe a sentence that if, let's say, there is no acknowledgement of maintainers in 7 or 14 days, they should use `/ptal` command. wdyt?

Reply:
Yeah `/ptal` can be a good option for the author of the issue. Kindly add this information in the doc.